src/Plugin/Action/SaveMedia.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\media_entity\Plugin\Action\SaveNode.
+ */
+
+namespace Drupal\media_entity\Plugin\Action;
+
+use Drupal\Core\Action\ActionBase;
+use Drupal\Core\Session\AccountInterface;
+
+/**
+ * Provides an action that can save any entity.
+ *
+ * @Action(
+ *   id = "media_save_action",
+ *   label = @Translation("Save media"),
+ *   type = "media"
+ * )
+ */
+class SaveMedia extends ActionBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function execute($entity = NULL) {
+    // We need to change at least one value, otherwise the changed timestamp
+    // will not be updated.
+    $entity->changed = 0;
+    $entity->save();
```

Comment:
@Berdir In the past, I've used the 'Save' views bulk operation to run code that is triggered upon node/entity save, like code in entity-save processing and entity save hooks. That way you don't have to make a VBO or a batch process to trigger some code that already runs on entity save. I'm not sure if that's the same logic that node module has for having the action. I personally would rather keep it, but whatever you think!

I had copied over the publish/unpublish actions into media_entity, but then I scrapped them before committing the code. I was wondering what media_entity's approach is for the published status of media entities. I mostly wonder because as far as I know, there is not a UI element to set a media entity to published or unpublished. They are just published by default, at least when they are created in the standard MediaForm. If they are created elsewhere, they are unpublished by default. So I thought maybe media entities are designed to almost always be published? Either way let me know, as I can easily add those two actions back in.